orchestrator/test/conftest.py

```diff
@@ -125,3 +127,23 @@ def in_tmp_dir(tmp_path: Path) -> Path:
     os.chdir(tmp_path)
     yield tmp_path
     os.chdir(cwd)
+
+
+@pytest.fixture()
+def app_setup(request, device: Device, support_app: str, support_test_app: str, test_butler_service: str
+              ) -> Tuple[Application, TestApplication, ServiceApplication]:
+    uninstall_apk(support_app, device)
```

Comment:
What part?

Reply:
the "uninstall_apk"

Reply:
uninstall_apk

Reply:
Oh, I added it just in case an uninstall fails in some other test that doesn't use the fixture. It's basically making sure we start from a clean state.